test/firtool/pass-plugins.fir

```diff
@@ -0,0 +1,19 @@
+; RUN: firtool -mlir-timing -initial-firrtl-pass-plugin='strip-debuginfo' \ 
+; RUN:   -high-firrtl-pass-plugin='strip-debuginfo' \
```

Comment:
Would be nice if this test used different passes for each to ensure the right is inserted in correct place, but that requires multiple such passes so only if that's easy to alter.

Reply:
Thanks, I'll add this test :)